x-pack/plugins/security_solution/public/common/components/navigation/solution_grouped_nav/solution_grouped_nav_panel.styles.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiPanel } from '@elastic/eui';
+import styled from 'styled-components';
+
+export const EuiPanelStyled = styled(EuiPanel)<{ $hasBottomBar: boolean }>`
+  position: fixed;
+  top: 95px;
+  left: 247px;
```

Comment:
This is probably ok for now, but isn't especially maintainable in case the components these values are based off of change. I would suggest finding the variable equivalent to the header offset and solution nav width to use here.

Reply:
I agree, but I couldn't find them. I'll leave it this way in this PR, we can talk about it offline